harbor-helm/values.yaml

```diff
@@ -648,11 +648,11 @@ database:
     # set the service account to be used, default if left empty
     serviceAccountName: ""
     image:
-      repository: goharbor/harbor-db
-      tag: dev
+      repository: registry.suse.de/devel/caps/registry/containers/registry/harbor-db
+      tag: 12.2
     # the image used by the init container
     initContainerImage:
-      repository: busybox
+      repository: registry.suse.com/suse/sle15
       tag: latest
```

Comment:
I would use `sp2` as a tag here, to align with the base images used by other components

Reply:
Hm, isn't the latest tag ensuring that it indeed is SP2 image and not SLE15GA?

Reply:
registry.suse.com/suse/sle15:latest is actually SP1. I do not know if there's an image with SP2...